vignettes/howtogenefilter.Rmd

```diff
@@ -0,0 +1,139 @@
+---
+title: "Using the genefilter function to filter genes from a microarray dataset"
+author:
+- name: Khadijah Amusat
+  affiliation: Vignette translation from Sweave to R Markdown / HTML
+date: `r format(Sys.time(), '%B %d , %Y')`
+output:
+  BiocStyle:html_documents:
```

Comment:
Line 8 should be `BiocStyle::html_document:`, where there are 3 colons total and document is without an s.